cookbook/request/load_balancer_reverse_proxy.rst

```diff
@@ -57,9 +62,9 @@ the IP address ``192.0.0.1`` or matches the range of IP addresses that use
 the CIDR notation ``10.0.0.0/8``. For more details, see the
 :ref:`framework.trusted_proxies <reference-framework-trusted-proxies>` option.
 
-That's it! Symfony will now look for the correct ``X-Forwarded-*`` headers
-to get information like the client's IP address, host, port and whether or
-not the request is using HTTPS.
+That's it! Symfony will now look for the correct headers to get information
+like the client's IP address, host, port and whether or not the request is
```

Comment:
correct, he said the same. The only case where it is needed is when explicitly we want to emphasize main one thing. In this case however it is a bunch of things and it is not in the main part of the sentence.